flang/lib/Semantics/check-omp-structure.cpp

```diff
@@ -4550,6 +4550,97 @@ void OmpStructureChecker::Leave(const parser::DoConstruct &x) {
   Base::Leave(x);
 }
 
+void OmpStructureChecker::Enter(const parser::OpenMPInteropConstruct &x) {
+  bool isDependClauseOccured{false};
+  int targetCount{0}, targetSyncCount{0};
+  const auto &dir{std::get<parser::Verbatim>(x.t)};
+  std::list<std::string> ObjectNameList;
+  PushContextAndClauseSets(dir.source, llvm::omp::Directive::OMPD_interop);
+  const auto &clauseList{std::get<parser::OmpClauseList>(x.t)};
+  for (const auto &clause : clauseList.v) {
+    common::visit(
+      common::visitors{
+        [&](const parser::OmpClause::Init &InitClause) {
+          const auto &InteropTypeList{
+              std::get<parser::OmpInitClause::InteropTypes>(InitClause.v.t)};
+          for (auto &InteropTypeVal : InteropTypeList.v) {
+            if (*(parser::Unwrap<parser::InteropType::Kind>(
+                    InteropTypeVal)) ==
+                parser::InteropType::Kind::TargetSync) {
+              ++targetSyncCount;
+            } else {
+              ++targetCount;
+            }
+            if (targetCount > 1 || targetSyncCount > 1) {
+              context_.Say(GetContext().directiveSource,
+                          "Each interop-type may be speciﬁed "
+                          "at most once."_err_en_US);
```

Comment:
Recently, Kparzysz suggested keeping the Error message in a single line.
Something like:
```suggestion
                          "Each interop-type may be speciﬁed at most once."_err_en_US);
```
git clang-format won't complain about it.
Do the same for all the other error messages as well.

Reply:
Sure, will do it. Thanks!